content/en/docs/tasks/debug/debug-cluster/kubectl-node-debug.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Debugging Kubernetes nodes with kubectl
+content_type: task
+---
+
+<!-- overview -->
+This page shows how to debug a [node](/docs/concepts/architecture/nodes/) running on the Kubernetes cluster using `kubectl debug` command.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Debugging Node using kubectl debug node
+
+Use the `kubectl debug node` command to deploy a Pod to a Node that you want to troubleshoot.
+This command is helpful in scenarios where you can't access your Node by using an SSH connection.
+When the Pod is created, the Pod opens a interactive shell on the Node.
+To create an interactive shell on a Node using `kubectl debug`, run:
+
+```shell
+kubectl debug node/mynode -it --image=ubuntu
+```
+
+```shell
+Creating debugging pod node-debugger-mynode-pdx84 with container debugger on node mynode.
+If you don't see a command prompt, try pressing enter.
+root@ek8s:/#
+```
+
+Run debug commands to help you gather information and troubleshoot issues. Commands 
+that you might use to debug, such as `ip`, `ifconfig`, `nc`, `ping`, and `ps`. You can also
+install other tools, such as `mtr`, `tcpdump`, and `curl`, from the respective package manager.
+
+Get the following logs from the shell session to look for issues on the Node. Note that the root filesystem
+of the Node will be mounted at `/host`
+
+* `/host/var/log/kubelet.log` - logs from the `kubelet`, responsible for running containers on the node
+* `/host/var/log/kube-proxy.log` - logs from `kube-proxy`, which is responsible for directing traffic to Service endpoints
+* `/host/var/log/containerd.log` - logs from `containerd` process running on node
+* `/host/var/log/syslog` - shows general messages and info regarding the system
+* `/host/var/log/kern.log` - shows kernel logs
+
+When creating a debugging session on a Node, keep in mind that:
+
+* `kubectl debug` automatically generates the name of the new pod based on
+  the name of the node.
+* The root filesystem of the Node will be mounted at `/host`.
+* The container runs in the host IPC, Network, and PID namespaces, although
+  the pod isn't privileged, so reading some process information may fail,
+  and `chroot /host` will fail.
```

Comment:
```suggestion
* The container runs in the host's IPC, network, and PID namespaces, although
  the debug pod isn't privileged. This means that so reading some process information may
  fail (because access to that information is restricted to superusers),
  and `chroot /host` will fail.
```

Reply:
```suggestion
* Although the container runs in the host IPC, Network, and PID namespaces,
  the pod isn't privileged. This means that reading some process information might fail
  because access to that information is restricted to superusers. `chroot /host` will fail.
```